app/assets/stylesheets/ama_layout/layout/helper-classes.scss

```diff
@@ -533,6 +533,70 @@ button[disabled],
   word-break: break-word;
 }
 
+.semi-bold{
+  font-weight: 600;
+
+  &--small{
+    @extend .semi-bold;
+    font-size: $base-font-size*0.75;
+  }
+
+  &--medium{
+    @extend .semi-bold;
+    font-size: $base-font-size*1.3;
+  }
+
+  &--large{
+    @extend .semi-bold;
+    font-size: $base-font-size*1.85;
+  }
+
+  &--x-large{
+    @extend .semi-bold;
+    font-size: $base-font-size*3;
+  }
+}
+
+.bold{
+  font-weight: 700;
+
+  &--small{
+    @extend .bold;
+    font-size: $base-font-size*0.75;
+  }
+
+  &--medium{
+    @extend .bold;
+    font-size: $base-font-size*1.3;
+  }
+
+  &--large{
+    @extend .bold;
+    font-size: $base-font-size*1.85;
+  }
+
+  &--x-large{
+    @extend .bold;
+    font-size: $base-font-size*3;
+
+    &--green{
+      @extend .bold--x-large;
+      color: $brand-green
+    }
+
+    &--no-lineheight{
+      @extend .bold--x-large;
+      line-height: 0;
+      margin-bottom: $base-margin*1.65;
+      margin-top: $base-margin*1.65;
+    }
+
+    sup{
```

Comment:
does this need to live inside bold Xl text since it's not extending it? or can it live outside - ie can we have semi-bold superscript, or regular superscript?